### PR TITLE
IRO-1229 - minor copy edit on careers

### PR DESCRIPTION
**Pull request by brekk**

### CODE REVIEW

**Review by YuryStaravoitav — APPROVED** [review]
APPROVED

File: src/pages/careers.js [hunk 1]
```diff
@@ -280,11 +280,11 @@ function Careers() {
...
                 <p>
-                  We want you to take the time off you need to relax and
-                  recharge. In addition to federal holidays, we offer accrued
-                  time off of up to 15 days per calendar year, we close the
-                  office for winter break, and sick time is accrued at up to 8
-                  days per calendar year.
+                  We want you to take the time you need to relax and recharge.
+                  In addition to 12 federal holidays and an office closure for
+                  winter break, we offer a generous time-off policy. For
+                  salaried employees, we have a flexible vacation and sick days,
```
Comment (dguenther) [inline src/pages/careers.js:286]:
```suggestion
                  salaried employees, we have flexible vacation and sick days,
```
This is a bit grammatically ambiguous, not sure which is right

**Review by dguenther — APPROVED** [review]
Looks good to me, let's run it by @leanthebean

File: src/pages/careers.js (hunk 1 above)
Comment (deekerno) [reply src/pages/careers.js:286]: The former indicates (to me, at least) that there is a singular vacation that everyone takes while the latter communicates that employees have a number of vacation days that they can take at their convenience.

**Pull request merged**